packages/augur-ui/src/modules/global-chat/components/global-chat.tsx

```diff
@@ -1,19 +1,29 @@
-import React, { useState } from 'react';
+import React, { useState, Suspense, lazy } from 'react';
 import Styles from 'modules/global-chat/components/global-chat.styles.less';
 import { SecondaryButton } from 'modules/common/buttons';
 import { useAppStatusStore } from 'modules/app/store/app-status';
 import { ThickChevron, Close } from 'modules/common/icons';
 import classNames from 'classnames';
+import { useAppStatus } from 'modules/app/store/app-status-hooks';
+
+const ThreeBoxChat = lazy(() =>
+  import(/* webpackChunkName: '3box-chat' */ 'modules/global-chat/components/three-box-chat')
+);
 
 export const GlobalChat = () => {
-  let { env } = useAppStatusStore();
+  let { isLogged, loginAccount, env, initialized3box } = useAppStatusStore();
   const [show, setShow] = useState(false);
+  const signer = loginAccount.meta?.signer;
```

Comment:
for this we can actually just pull this out from the context so above `{ loginAccount: { meta: { signer } }, isLogged, etc...`

this way signer will just be the signer or it's undefined which satisfies the provider assignment below. since we never use loginAccount or Meta we should just pull directly to the important part.